templates/zerver/emails/new_login/new_login_alert.txt

```diff
@@ -0,0 +1,18 @@
+{# Informs user of a new login to their account #}
+
+Hello, {{ user.short_name | title }}.
+
+This is a notification that a new login to your Zulip account has just occured.
+
+Details
+Account: {{ user.email }}
+Date/Time: {{ device_info.login_time }}
+Device: {{ device_info.device_browser | title }} on {{ device_info.device_os | title }}
```

Comment:
I guess we don't need the `| title` if we set the case correctly as I suggest below.